ingest/jobs/Get_Ingest_Config/tests/end_to_end/features/fixtures.py

```diff
@@ -0,0 +1,30 @@
+from azure.identity import DefaultAzureCredential
+from azure.storage.filedatalake import DataLakeServiceClient
+from behave import fixture
+from constants import ADLS_URL, RAW_CONTAINER_NAME, AUTOMATED_TEST_OUTPUT_DIRECTORY_PREFIX
+from utils.azure.adls import filter_directory_paths_adls, delete_directories_adls
+
+@fixture
+def azure_adls_clean_up(context, ingest_directory_name: str):
+    credential = DefaultAzureCredential()
+    adls_client = DataLakeServiceClient(account_url=ADLS_URL, credential=credential)
```

Comment:
I think it would be better to have these credentials set in the context.

Reply:
@lorraine-houston-ensono please review this as well.

Reply:
Can you explain why? Again this is from preexisting code in the source.

Reply:
This coupling of client creation and the function seems like not the best idea to me. And if we have more functions that use this client, we will duplicate this code. But the context might not be the best place for it either. I'll resolve, cause I'm not sure what the best solution would be.